test/quantization/core/test_quantized_op.py

```diff
@@ -4687,7 +4709,7 @@ def _test_qconv_impl(
            Y_scale=st.floats(4.2, 5.6),
            Y_zero_point=st.integers(0, 4),
            use_bias=st.booleans(),
-           use_relu=st.booleans(),
+           post_op=st.sampled_from(["none", "relu"]),
```

Comment:
can be a separate PR, but might make sense to split the conv and conv_relu test as well

Reply:
Thanks for the suggestions, I have split the conv and conv_relu test and so as the other similar test cases.